okhttp/src/main/java/okhttp3/internal/Transmitter.java

```diff
@@ -214,8 +230,8 @@ public boolean supportsUrl(HttpUrl url) {
     return streamAllocation.connection().supportsUrl(url);
   }
 
-  public void acquire(RealConnection connection, boolean reportedAcquired) {
-    streamAllocation.acquire(connection, reportedAcquired);
+  public void transmitterAcquireConnection(RealConnection connection, boolean reportedAcquired) {
```

Comment:
you prefixed it with `transmitter` for not to be confused with `streamAllocation`? Otherwise, since the class is `Transmitter` already, let's remove the prefix?

Reply:
Fixed!